suspect_resolutions/commit_correlation.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from sentry.models import CommitFileChange, GroupRelease, ReleaseCommit
+
+
+def is_issue_commit_correlated(resolved_issue_id: int, candidate_issue_id: int) -> bool:
+    # test whether the resolved_issue commit data is related to the suspect issue commit data
+    # check set intersection between set of files touched by one issue with set of files touched by another issue
+    # check line changes to make it more accurate
+
+    resolved_issue_files = {filename for filename in (get_files_changed(resolved_issue_id))}
+    candidate_suspect_resolution_files = {
+        filename for filename in (get_files_changed(candidate_issue_id))
+    }
+
+    return not resolved_issue_files.isdisjoint(candidate_suspect_resolution_files)
+
+
+def get_files_changed(issue_id: int):
```

Comment:
I'm thinking about this some more and should this method have a start and stop time-window? 

Consider the fact that an issue can be observed over many releases, that means for a given issue there could be multiple `GroupRelease`s. Would it make sense to do a query-range for all `Releases` associated with the issue and check if the files changed?